features/api/ideas_workshop_ideas.feature

```diff
@@ -869,13 +864,12 @@ Feature:
 
   Scenario: As a logged-in user I can publish my idea in the status DRAFT
```

Comment:
Please, change also this scenario description.

Reply:
done tnx